qcodes/dataset/data_set.py

```diff
@@ -890,6 +892,21 @@ def get_data(self,
         return get_data(self.conn, self.table_name, valid_param_names,
                         start, end)
 
+    def cache_parameter_data(self):
```

Comment:
We came across a situation where we wanted to get access to the datasaver dataset during measurement loops, and flushing every loop helped with at least having it readily available. Is the intention here to have an easily accessible container for the data, without too much effort? This is Mason by the way 👍

Reply:
@hbarovertwo This tries to do two things but only one of them is implemented at the moment. I think I forgot to mention that. 

1) Make it possible to read the data gradually from disk as it is beeing written in a different process.  
2) Automatically keep a cache of the data in the process that is writing the the data. This is not currently implemented. 
   That cache can be used for inprocess liveplot or send over a socked or some such thing.